Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -939,14 +939,15 @@ private class ParameterSetNames
         }
 
         /// <summary>
-        /// A Google Cloud Storage object description 
+        /// A Google Cloud Storage object description of the object to read from. They can be retrieved using
```

Comment:
"object description of the object to read from" sounds confusing. How about just

"Cloud Storage object to read from. Can be obtained using Get-GcsObject or Find-GcsObject."